test/README.md

```diff
@@ -0,0 +1,147 @@
+# Testing, Testing
+
+## Running Tests
+
+```console
+$ ./run.sh
+
+usage: run.sh [-t test ...] [imageTag ...]
+   ie: run.sh debian:wheezy
+       run.sh -t utc python:3-onbuild
+       run.sh -t utc python:3-onbuild -t py-onbuild
+
+This script processes the specified Docker images to test their running
+environments.
+$ ./run.sh debian:wheezy
+testing debian:wheezy
+	'utc' [1/2]...passed
+	'cve-2014--shellshock' [2/2]...passed
+$ ./run.sh -t python-hy debian:wheezy
+testing debian:wheezy
+$ ./run.sh -t utc python:3 python:2
+testing python:3
+	'utc' [1/1]...passed
+testing python:2
+	'utc' [1/1]...passed
+$ ./run.sh -t utc -t python-hy python:3 python:2
+testing python:3
+	'utc' [1/2]...passed
+	'python-hy' [2/2]...passed
+testing python:2
+	'utc' [1/2]...passed
+	'python-hy' [2/2]...passed
```

Comment:
Maybe throw in `ruby:onbuild` and `jruby:onbuild` example together to show exclude on an aliased version or would that be too many examples?